api/models/site-build-task.js

```diff
@@ -8,6 +11,59 @@ const associate = ({ BuildTaskType, Site, SiteBuildTask }) => {
   });
 };
 
+async function startBuildTask(buildId) {
+  const siteBuildTask = this;
+  const {
+    BuildTask,
+    BuildTaskType,
+    Build,
+    Site,
+  } = siteBuildTask.sequelize.models;
+
+  const buildTask = await BuildTask.create({
+    buildTaskTypeId: siteBuildTask.buildTaskTypeId,
+    buildId,
+    name: `build: ${buildId}, type: ${siteBuildTask.buildTaskTypeId}`,
+  });
+
+  // TODO: why
+  const createdBuildTask = await BuildTask.findByPk(buildTask.id, {
+    include: [
+      { model: BuildTaskType, required: true },
+      { model: Build, required: true, include: [{ model: Site, required: true }] },
+    ],
+  });
+
+  BuildTaskQueue.sendTaskMessage(createdBuildTask);
+}
+
+// TODO: should this be converted to sequelize scope?
+async function findTasksToStart({ build, startsWhen }) {
```

Comment:
Should we nest or call this  method in `startBuildTask` to simplify how we start tasks from other parts of the app?